test/elixir/test/test_helper.exs

```diff
@@ -1,3 +1,5 @@
+# credo:disable-for-this-file Credo.Check.Consistency.ParameterPatternMatching
```

Comment:
Why wouldn't we disable this globally, or fix the problem in this one file? Keep in mind I am new to Elixir, but not to linting and formatting...exceptions in a single file are usually bad news.

Reply:
@dottorblaster This still needs addressing before I can approve this PR.

Reply:
Done! Sorry, I missed it 🙇